code/game/objects/items/weapons/policetape.dm

```diff
@@ -261,10 +261,11 @@ var/global/list/image/hazard_overlays //Cached hazard floor overlays for the bar
 	neighbors = 0
 	for (var/look_dir in global.cardinal)
 		var/turf/target_turf = get_step(src, look_dir)
-		var/obj/structure/tape_barricade/B = locate(/obj/structure/tape_barricade, target_turf)
-		//We connect to walls and other tape_barricades
-		if((B && !QDELETED(B)) || (!B && target_turf?.is_wall()))
-			neighbors |= look_dir
+		if(target_turf)
+			var/obj/structure/tape_barricade/B = locate(/obj/structure/tape_barricade, target_turf)
```

Comment:
```suggestion
			var/obj/structure/tape_barricade/B = locate(/obj/structure/tape_barricade) in target_turf
```
uhhhh. i don't know if the original syntax is valid or not but it definitely gives me the heebie-jeebies. please change it and i will get around to changing the other cases of this also